helm/openwhisk/README.md

```diff
@@ -0,0 +1,178 @@
+* The Kubernetes cluster supports Dynamic Volume Provisioning and has a default StorageClass defined with an associated provisioner.
+* The Kubernetes cluster supports Dynamic Volume Provisioning and when the chart is deployed, the value `k8s.persistence.defaultStorageClass` is set to a StorageClass which has an associated provisioner.
+
+### PodSecurityPolicy Requirements
+
+OpenWhisk's Invokers need elevated security permissions to be able to create the containers that execute the user actions. Therefore, this chart requires a PodSecurityPolicy that permits host access to be bound to the target namespace prior to installation.  If the default Pod security policy on your cluster is not restrictive then this step is not needed. If the default is restrictive, please create a new namespace with either a predefined PodSecurityPolicy `ibm-anyuid-hostpath-psp`:
+
+* Predefined PodSecurityPolicy name: [`ibm-anyuid-hostpath-psp`](https://ibm.biz/cpkspec-psp)
+
+Alternatively, you can have your cluster administrator setup a custom PodSecurityPolicy for you using the below definition:
+
+* Custom PodSecurityPolicy definition:
+
+    ```
+    apiVersion: extensions/v1beta1
+    kind: PodSecurityPolicy
+    metadata:
+        name: ibm-anyuid-hostpath-psp
+    annotations:
+        kubernetes.io/description: "This policy allows pods to run with 
+        any UID and GID and any volume, including the host path.
+        WARNING:  This policy allows hostPath volumes.
+        Use with caution." 
+    spec:
+        allowPrivilegeEscalation: true
+        fsGroup:
+            rule: RunAsAny
+        requiredDropCapabilities: 
+        - MKNOD
+        allowedCapabilities:
+        - SETPCAP
+        - AUDIT_WRITE
+        - CHOWN
+        - NET_RAW
+        - DAC_OVERRIDE
+        - FOWNER
+        - FSETID
+        - KILL
+        - SETUID
+        - SETGID
+        - NET_BIND_SERVICE
+        - SYS_CHROOT
+        - SETFCAP 
+        runAsUser:
+            rule: RunAsAny
+        seLinux:
+            rule: RunAsAny
+        supplementalGroups:
+            rule: RunAsAny
+        volumes:
+        - '*'
+    ```
+
+## Initial setup
+
+Identify the Kubernetes worker nodes that should be used to execute
+user containers.  Do this by labeling each node with
+`openwhisk-role=invoker`.  If you have a multi-node cluster, for each node <INVOKER_NODE_NAME>
+you want to be an invoker, execute
+```shell
+kubectl label nodes <INVOKER_NODE_NAME> openwhisk-role=invoker
+```
+For a single node cluster, simply do
+```shell
+kubectl label nodes --all openwhisk-role=invoker
+```
+
+## Installing the Chart
+
+Please ensure that you have reviewed the [prerequisites](#prerequisites) and the [initial setup](#initial-setup) instructions.
+
+To install the chart using helm cli:
+
+```bash
+$ helm install --tls community/openwhisk --namespace <my-namespace> --name <my-release> --set whisk.ingress.apiHostName=<cluster-ip-address>
```

Comment:
Should we say `openwhisk` as the chart name here and replace with `community/openwhisk` in the downstream?

Reply:
Yes, this was my oversight, we should keep the chart name 'openwhisk'